parlai/core/pytorch_data_teacher.py

```diff
@@ -23,22 +24,14 @@
 from functools import wraps
 import importlib
 from functools import lru_cache
-
-try:
-    import torch  # noqa: F401
-except ImportError:
-    raise ImportError('Need to install Pytorch: go to pytorch.org')
+import torch  # noqa: F401
 from torch.utils.data import ConcatDataset, Dataset, DataLoader, sampler
 from torch.multiprocessing import Lock, Value
 import ctypes
 from threading import Thread, Condition, RLock
 
-
-if torch.version.__version__.startswith('0.'):
-    raise ImportError(
-        "Please upgrade to PyTorch >=1.0; "
-        "visit https://pytorch.org for instructions."
-    )
+# Perform Torch version check
+check_torch_version()
```

Comment:
we should perform check prior to importing, as you have done in other files

Reply:
Done so in latest commit!